collector/settings.mdx

```diff
@@ -141,7 +141,7 @@ helper functions are expected to be defined, and the one we connect to for serve
     <tr>
       <td>db_sslrootcert (<code>DB_SSLROOTCERT</code>)</td>
       <td>system certificate store</td>
-      <td>Path to SSL certificate authority (CA) certificate(s) to use to verify the server's certificate, or one of <code>rds-ca-2015-root</code> or <code>rds-ca-2019-root</code> to use the built-in AWS RDS certificates</td>
+      <td>Path to SSL certificate authority (CA) certificate(s) to use to verify the server's certificate, or <code>rds-ca-global</code> to use the built-in AWS RDS certificates</td>
```

Comment:
Also, we should adjust the RDS setup instructions to use "rds-ca-global" instead of "rds-ca-2019-root" (since the 2019 CA is included in the global bundle, its safe to just replace it everywhere)